contracts/Swaplace.sol

```diff
@@ -126,6 +140,14 @@ contract Swaplace is SwapFactory, ISwaplace, IERC165 {
    * @dev Getter function for _totalSwaps.
    */
   function totalSwaps() public view returns (uint256) {
-    return _totalSwaps;
+    uint256 swapId;
+
```

Comment:
totalSwaps is never called on-chain, therefore there is no gas optm., we can keep the original way to preserve legibility